tests/test_bolt2-01-open_channel.py

```diff
@@ -342,6 +343,7 @@ def test_open_channel_opener_side_wrong_announcement_signatures(runner: Runner)
         #
         # So we should change the OneOf to all exception and stop when
         # the first one succided
-        ExpectDisconnect(),
+        Sequence(ExpectDisconnect(), enable=is_cln),
+        Sequence(ExpectMsg("error"), enable=(not is_cln)),
```

Comment:
I have some questions.
1. Why are we using `Sequence` instead of `OneOf`? Shouldn't this be a `OneOf` list of acceptable responses? That way we might not have to specify `is_cln`?
2. Instead of `ExpectMsg("error")` maybe we can use `ExpectError()`.

I was going through the BOLT specification and found 
```
The recipient:
if signature is incorrect OR non-compliant with LOW-S-standard rulehttps://github.com/bitcoin/bitcoin/pull/6769:
MUST send a warning and close the connection, or send an error and fail the channel.
```
We are checking for Disconnect(closing connection) and Error, maybe we should also check channel closure?

Reply:
Ah probably the `ExpectDisconnect` when find the connection ready, and it is able to consume the buffer can stash the info inside the Runner, and then `ExpectMsg` check for stashed message :/ but this is so complicated.

We can track this with an issue? In this way we can track it?